dev-tools/ecs-migration.yml

```diff
@@ -625,3 +625,8 @@
   to: event.dataset
   alias: false
   comment: No alias mapping as field did not always exist
+
+- from: http.url
```

Comment:
I would also expect a change to the fields.yml file that introduces this alias?

Also in the code I couldn't find where `http.url` is removed?

Reply:
Right now it's just being duplicated and we're relying on ECS to provide that field as a non-alias.

Reply:
If you're going to keep the original `http.url` field present and just copy its value, you should remove the entry from `ecs-migration.yml`.

Reply:
It should still show up here if we potentially need to migrate this in the dashboards.

But in case we keep the old field around, `alias: true` should be set to false as we can't create an alias for the old field.